.github/workflows/raise-pr.yml

```diff
@@ -31,19 +31,16 @@ jobs:
       - name: checkout-the-move2kube-fork-of-upstream-repo
         run: |
           git clone 'https://move2kube:${{ secrets.MOVE2KUBE_PATOKEN }}@github.com/move2kube/community-operators'
-          cd community-operators/
+          cd community-operators/ || exit 1
```

Comment:
Is this or condition meant to exit with error the directory is not available?

Reply:
yes